test/aio5gc/lib/tools/tools.go

```diff
@@ -46,9 +46,9 @@ func GenerateDefaultConf(controlIF config.ControlIF, dataIF config.DataIF, amf c
 				Nia2: true,
 			},
 			Ciphering: config.Ciphering{
-				Nea0: false,
+				Nea0: true,
```

Comment:
please keep NEA0 and NEA2 both at true
that way PacketRusher will work with 5G Core that only work with NEA0 by default (free5gc) and proprietary cores that mandates a secure ciphering algorithms by default